### PR TITLE
Added an index in file walkthrough_apiv1.md

### PR DESCRIPTION
**Pull request by raconte**
Included an index for the file: /doc/manuals/user/walkthrough_apiv1.md

### CODE REVIEW

File: doc/manuals/user/walkthrough_apiv1.md [hunk 1]
```diff
@@ -23,7 +62,7 @@ information). Some remarks to take into account to use this stuff:
 -   Note that each main section is divided in two sub-sections: the
     first being about standard operations while the second is about
     convenience operations. In fact, each sub-section is an independent
-    tutorial (for a total of 4 tutorials summing up both sections) that
+    tutorial (for efore starting...a total of 4 tutorials summing up both sections) that
```
Comment (fgalan) [inline doc/manuals/user/walkthrough_apiv1.md]: Copy paste typo? ;)

**Comment by raconte** [comment]
Fixed in a080502

File: doc/manuals/user/walkthrough_apiv1.md (hunk 1 above)
Comment (raconte) [inline doc/manuals/user/walkthrough_apiv1.md]: Fixed in a080502

**Comment by fgalan** [comment]
LGTM

**Pull request merged**